src/name_bazaar/ui/pages/user_bids_page.cljs

```diff
@@ -57,46 +58,50 @@
             :on-change #(dispatch [:offerings.user-bids/set-params-and-search {:min-end-time-now? (not (aget %2 "checked"))}])}]]]))))
 
 (defn user-bids []
-  (let [search-results (subscribe [:offerings/user-bids])]
+  (let [search-results (subscribe [:offerings/user-bids])
+        route-params (subscribe [:district0x/route-params])]
     (fn [{:keys [:title :no-items-text]}]
       (let [{:keys [:items :loading? :params :total-count]} @search-results]
-        [app-layout
-         [ui/Segment
-          [ui/Grid
-           {:padded true
-            :class "no-inner-horizontal-padding mobile-inner-vertical-padding"}
-           [ui/GridColumn
-            {:width 16
-             :class :join-upper}
-            [:h1.ui.header title]]
-           [ui/GridColumn
-            {:computer 8
-             :tablet 16
-             :mobile 16}
-            [user-bids-search-checkboxes]]
-           [ui/GridColumn
-            {:computer 6
-             :tablet 8
-             :mobile 16
-             :floated "right"
-             :vertical-align :bottom}
-            [user-bids-order-by-select]]]
-          [offering-infinite-list
-           {:class "primary"
-            :total-count total-count
-            :offset (:offset params)
-            :loading? loading?
-            :no-items-text no-items-text
-            :on-next-load (fn [offset limit]
-                            (dispatch [:offerings.user-bids/set-params-and-search
-                                       {:offset offset :limit limit} {:append? true}]))}
-           (doall
-             (for [[i offering] (medley/indexed items)]
-               [offering-list-item
-                {:key i
-                 :offering offering
-                 :header-props {:show-auction-winning? true
-                                :show-auction-pending-returns? true}}]))]]]))))
+        [:div.wrapper
+         [name-bazaar-meta-tags {:title title :description (if-let [address (:user/address @route-params)]
```

Comment:
:title "NameBazaar - [title]"
:description "See all bids for ENS name offerings of [address]"